src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala

```diff
@@ -33,20 +33,20 @@ class ClaimSimulation extends Simulation {
   private val today: LocalDate = LocalDate.now()
 
   private val random = new Random
-  private val ninoFormat = getNinoFormat
+  private val fourDigitFormat = getFourDigitFormat
 
-  private val dateAYearAgo: LocalDate = LocalDate.now().minusYears(1)
+  private val dobUnderOne: LocalDate = LocalDate.now.minusMonths(6).withDayOfMonth(1)
```

Comment:
Can't import any dependencies here :( - so can't use the common code

Reply:
Is there no way to import a java dependency into Scala?

Reply:
It's a gatling issue rather than Scala. Gatling is a standalone binary that uses Scala as a DSL, so doesn't behave exactly like a standard Scala program.